docs/data/data-grid/row-grouping/row-grouping.md

```diff
@@ -323,36 +338,34 @@ type GridRowSelectionPropagation = {
 };
 ```
 
-When `rowSelectionPropagation.descendants` is set to `true`.
+### Row selection propagation
```

Comment:
I'd keep this under the same top level heading to avoid confusion about this something different than "automatic p.... selection"
```suggestion
```